### DISCUSSION TITLE
API usage

### DISCUSSION THREAD

**Discussion by Donvine254** [tosdr/API#2] [Q&A] [answered]
there is little to no documentation on how to use the API key to search for website ratings. If anyone knows how to create a query that will return specific info about a searched website such as facebook, kindly help

**Comment by shadowwwind** [comment]
how I understood it you would use the search API to find a service and then use the service API to get more detailed information. 
this are the docs https://developers.tosdr.org/dev/ 
and the service API seams to have more detail here https://github.com/tosdr/API/tree/master/functions/Service/GET/v1

**Comment by Donvine254** [comment]
Thank you for the help, but i do not fully understand since there is no example or a list of services u can fetch

**Reply by JustinBack** [reply]
https://api.tosdr.org/service/v1 - Lists all services in a pagination type of request.

**Comment by Donvine254** [comment]
Thank you Justin, can you also provide an API call to get reviews for a service such as instagram that will return the JSON data for the service?

**Reply by JustinBack** [reply, marked as answer]
You can lookup a service by searching for its slug, service name or url by using https://api.tosdr.org/search/v4/?query=instagram - Use the ID to lookup using the Service API or use the search result :-)

**Reply by JustinBack** [reply]
For example in case Instagram (ID: 219)

https://api.tosdr.org/service/v1/?service=219

**Comment by Donvine254** [comment]
more than helpful justin, thankyou
